backend/pkg/console/api_versions.go

```diff
@@ -29,23 +56,31 @@ type APIVersion struct {
 // versions. This will be used by the frontend to figure out what functionality is available
 // or should be rendered as not available.
 func (s *Service) GetAPIVersions(ctx context.Context) ([]APIVersion, error) {
-	versionsRes, err := s.kafkaSvc.GetAPIVersions(ctx)
+	cl, _, err := s.kafkaClientFactory.GetKafkaClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+	req := kmsg.NewApiVersionsRequest()
+	req.ClientSoftwareVersion = version.Version
+	req.ClientSoftwareName = "RPConsole"
+
+	versionsRes, err := req.RequestWith(ctx, cl)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get kafka api version: %w", err)
+		return nil, fmt.Errorf("failed to get kafka api vers: %w", err)
 	}
 
 	err = kerr.ErrorForCode(versionsRes.ErrorCode)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get kafka api version. Inner error: %w", err)
+		return nil, fmt.Errorf("failed to get kafka api vers. Inner error: %w", err)
 	}
 
 	versions := make([]APIVersion, len(versionsRes.ApiKeys))
-	for i, version := range versionsRes.ApiKeys {
+	for i, vers := range versionsRes.ApiKeys {
```

Comment:
I am not sure the benefit of this naming change?

Reply:
just to avoid the naming collision with the imported package named `version`